stubs/Controllers/UsersController.stub

```diff
@@ -2,71 +2,168 @@
 
 namespace App\Http\Controllers\Api\v1;
 
-use App\JsonApi\Users\Hydrator;
-use App\JsonApi\Users\Search;
-use App\Models\User;
+use CloudCreativity\JsonApi\Contracts\Http\Requests\RequestInterface as JsonApiRequest;
 use CloudCreativity\LaravelJsonApi\Http\Controllers\EloquentController;
-use App\JsonApi\Users\Request as UsersRequest;
+use App\Models\User;
+use App\JsonApi\Users\Hydrator;
 
 class UsersController extends EloquentController
 {
     /**
-     * EloquentController constructor.
-     *
-     * @param User $model
-     * @param Hydrator|null $hydrator
-     * @param Search|null $search
+     * @param User $user
+     * @param Hydrator $hydrator
      */
-    public function __construct(
-        User $model,
-        Hydrator $hydrator = null
-    ) {
-        parent::__construct($model, $hydrator);
-    }
-
-    protected function getRequestHandler()
+    public function __construct(User $user, Hydrator $hydrator)
     {
-        return UsersRequest::class;
+        parent::__construct($user, $hydrator);
     }
 
     /**
-     * @SWG\Get(path="/users",
-     *   tags={"User actions"},
-     *   summary="Filter users with limited per page quantity.",
-     *   description="Filter users",
+     * Display a listing of the resource.
+     *
+     * @SWG\Get(
+     *   path="/users",
+     *   tags={"Users"},
+     *   summary="get me",
+     *   description="get me",
```

Comment:
Does the method return only one logged user?

Reply:
yes